lib/manageiq/rpm_build/generate_ansible_venv.rb

```diff
@@ -46,7 +46,10 @@ def populate
       def install_python_system_packages
         where_am_i
 
-        shell_cmd("#{pip_versioned} install virtualenv psutil==5.6.6", "PYTHONDONTWRITEBYTECODE" => "1")
```

Comment:
We don't need the `PYTHONDONTWRITEBYTECODE` anymore?

Reply:
`--no-compile` does the same thing and is what we used elsewhere: https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-no-compile